content/community/governance/index.md

```diff
@@ -1,40 +1,45 @@
-
-## Galaxy Executive Board
-
-**Keywords:** Strategic emphasis, long term planning and direction
 
-The *[Galaxy Exec](/community/exec/)* is a group of principal investigators from around the world.  The first iteration of this was formed at the _Galaxy Directions Summit_ in 2019, which gathered together ~20 PIs from around the world.
-
-## Galaxy Global Steering Committee (GGSC)
-
-**Keywords:** Strategic emphasis, medium and long term planning, international opportunities
-
-The Galaxy Exec formed the *[Global Galaxy Steering Committee (GGSC)](/community/steering/)* to represent the interests of communities engaged with the project, and to implement efficient consultative project governance structures for that purpose.  This group was formed in early 2021.
+The Galaxy Project is a global open science collaboration, providing free access to shareable, reproducible, transparent,
+best practice data intensive analysis for all researchers.
 
+In support of this mission, project activities include maintaining source code, documentation, data analysis tools, training resources, analysis services, technical support, and an inclusive, self-governing community.
 
-## Galaxy Working Groups
+## Galaxy Technical Board (TCB)
 
-**Keywords:** Tactical emphasis, short and medium term planning, area/domain/component focused, applied
+The [TCB](/community/governance/gtb/) is a governance structure representing [Galaxy Working Groups](/community/wg/) (WGs) that manage
+different aspects of the Galaxy Ecosystem.
+The Galaxy Working Groups and the [TCB](/community/governance/gtb/) is focussed on technical development, manage important project infrastructure assets,
+including source code. WGs are self-organised, and nominate a representative, which serves on the [TCB](/community/governance/gtb/).
+The [GTB](/community/governance/gtb/) contributes to the roadmap, especially by providing technical input.
 
-*[Galaxy Working Groups](/community/wg/)* are community-driven small groups that manage different aspects of the Galaxy Ecosystem. Working groups set quarterly milestones for their areas, distribute work, and meet to discuss progress and roadblocks. Working Groups were launched in late 2020.
+Read more about [Galaxy Working Groups](/community/wg/).
 
+## Galaxy Community Board (GCB)
 
-## Committers Groups
+The [GCB](/community/governance/gcb/) is a governance structure representing *[Special Interest Groups](/community/sig/)* that are not primarily technical in nature, such as existing
+Proteomics and Microbial communities, engaging in capacity building, toolkit development and training.
+The [GCB](/community/governance/gcb/) will provide a supportive virtual forum
+for the exchange of ideas, and a governance body representing the interests of participating communities. It will organise and manage its own agenda and work,
```

Comment:
```suggestion
for the exchange of ideas. It will organise and manage its own agenda and work,
```
Otherwise we're repeating 'provide a governance structure' twice